File: VB/MyXtraGrid/Form1.vb
Imports System
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Windows.Forms

Namespace MyXtraGrid

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs)
            Dim dt As DataTable = New DataTable()
            dt.Columns.Add("ID", GetType(Integer))
            dt.Columns.Add("Name")
            dt.Columns.Add("Date", GetType(Date))
            For i As Integer = 0 To 15 - 1
                dt.Rows.Add(New Object() {i, String.Format("Name {0}", i), Date.Today.AddDays(i)})
            Next

            myGridControl1.DataSource = dt
        End Sub
    End Class
End Namespace

File: CS/MyXtraGrid/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace MyXtraGrid {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            DataTable dt = new DataTable();
            dt.Columns.Add("ID", typeof(int));
            dt.Columns.Add("Name");
            dt.Columns.Add("Date", typeof(DateTime));
            for(int i = 0;i < 15;i++) {
                dt.Rows.Add(new object[] { i, string.Format("Name {0}", i),DateTime.Today.AddDays(i) });
            }
            myGridControl1.DataSource = dt;
        }
    }
}